web/src/components/TaskElement.tsx

```diff
@@ -109,6 +111,17 @@ export const TaskElement = (props: Props): ReactElement => {
         {props.children}
         <Content>{props.task.summaryDescriptionMd || ""}</Content>
 
+        {hasBodyHeading && (
+          <>
+            <div>
+              <HorizontalLine />
+            </div>
+            <Heading level={2} styleVariant="h3">
```

Comment:
Design thought, not a critique of your work.

I'm curious how accurate these "variants" are. It seems to me that we're overriding more frequently than utilizing the default behavior. I only see a few instances in the codebase where we're actually using variant styling with the corresponding heading level. I get why we're doing it in code, but maybe there's an opportunity for us to rethink what these levels mean. Maybe this is a dumb thought.